cmd/icingadb-migrate/convert.go

```diff
@@ -701,30 +699,56 @@ func convertNotificationRows(
 //
 // [1]: https://github.com/Icinga/icinga2/blob/32c7f7730db154ba0dff5856a8985d125791c/lib/db_ido/dbevents.cpp#L1507-L1524
 // [2]: https://github.com/Icinga/icingadb/blob/8f31ac143875498797725adb9bfacf3d4/pkg/types/notification_type.go#L53-L61
-func convertNotificationType(notificationReason, state uint8) icingadbTypes.NotificationType {
+func convertNotificationType(notificationReason, state uint8) (string, error) {
```

Comment:
The function documentation is no longer accurate, especially the second part about the removed numeric representation in Icinga DB.